docs/source/markdown/podman-artifact-extract.1.md

```diff
@@ -0,0 +1,80 @@
+% podman-artifact-extract 1
+
+
+## WARNING: Experimental command
+*This command is considered experimental and still in development. Inputs, options, and outputs are all
+subject to change.*
+
+## NAME
+podman\-artifact\-extract - Extract an OCI artifact to a local path
+
+## SYNOPSIS
+**podman artifact extract** *artifact* *target*
+
+## DESCRIPTION
+
+Extract the blobs of an OCI artifact to a local file or directory. If the target
+path is a file or does not exists the artifact must either consists of one blob
+or if it has multiple blobs (layers) then the **--digest** or **--title** option
```

Comment:
`s/consists/consist/`

The `(layers)` explanation should probably happen with the _first_ use of the word “blob”.